### PR TITLE
[10.0][FIX] web_dialog_size: export modal

### PR DESCRIPTION
**Pull request by chienandalu**
- PR #733 introduced draggable dialogs wich broke some of them like the export data dialog.
- Also some style changes to follow OCA guidelines

cc @Tecnativa

### CODE REVIEW

File: web_dialog_size/__manifest__.py [hunk 1]
```diff
@@ -4,22 +4,19 @@
 
 {
     'name': "Web Dialog Size",
-
     'summary': """
         A module that lets the user expand a
         dialog box to the full screen width.""",
-
     'author': "ACSONE SA/NV, "
               "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
```
Comment (pedrobaeza) [inline web_dialog_size/__manifest__.py]: You can remove this old one

**Review by pedrobaeza — APPROVED** [review]

**Comment by chienandalu** [comment]
Mmm... got no runbot in CIs?

**Comment by pedrobaeza** [comment]
Uhm, strange indeed. @gurneyalex any clue?

File: web_dialog_size/static/src/js/web_dialog_size.js [hunk 2]
```diff
@@ -25,15 +25,16 @@ Dialog.include({
...
         });
         return res;
     },
 
     close: function() {
-        var draggable = this.$modal.draggable( "instance" );
+        var draggable = this.$modal.draggable('instance');
         if (draggable)
-            this.$modal.draggable("destroy");
+            this.$modal.draggable('destroy');
```
Comment (yajo) [inline web_dialog_size/static/src/js/web_dialog_size.js]: Leave this as it was please.

**Review by yajo — CHANGES_REQUESTED** [review]
Hmm this one is hard to approve without a runbot...

File: web_dialog_size/static/src/js/web_dialog_size.js (hunk 2 above)
Comment (pedrobaeza) [reply web_dialog_size/static/src/js/web_dialog_size.js]: I think this is because of the JS linter

Comment (chienandalu) [reply web_dialog_size/static/src/js/web_dialog_size.js]: @Yajo There was a mixin of single and double quotes. This edit was to respect the original module style. Is there any OCA preference to that respect?

Comment (yajo) [reply web_dialog_size/static/src/js/web_dialog_size.js]: Then the linter should be fixed.

Comment (yajo) [reply web_dialog_size/static/src/js/web_dialog_size.js]: OCA has no preference in that, so the best is not to change lines unless required. 😊

Comment (chienandalu) [reply web_dialog_size/static/src/js/web_dialog_size.js]: @Yajo Ok, but anyway isn't it better to not have them mixed?

Comment (yajo) [reply web_dialog_size/static/src/js/web_dialog_size.js]: Of course, I only mean to leave them as they are if you don't need to change them. 😉

**Comment by chienandalu** [comment]
@Yajo All right. Changes reduced to the minimum :)

**Comment by rousseldenis** [comment]
@chienandalu Are lint errors relevants ?

**Comment by taifu** [comment]
I used this PR in production code and it works (better than mine #774)

**Comment by chienandalu** [comment]
@rousseldenis Not to this fix as far as I can tell

**Review by rousseldenis — APPROVED** [review]
Tested on runbot

**Comment by pedrobaeza** [comment]
@Yajo please confirm for merging this

**Review by yajo — APPROVED** [review]

**Review by yajo — CHANGES_REQUESTED** [review]
Please obey [JS lints](https://travis-ci.org/OCA/web/jobs/291496379#L734-L741) except those that should not be there (see https://github.com/OCA/maintainer-quality-tools/issues/498).

**Comment by chienandalu** [comment]
@Yajo Linting fixed

**Review by yajo — APPROVED** [review]

**Pull request merged**